<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecoderUI</class>
 <widget class="QWidget" name="DecoderUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="8" column="2">
    <widget class="KLed" name="carrierLed">
     <property name="state">
      <enum>KLed::Off</enum>
     </property>
     <property name="look">
      <enum>KLed::Sunken</enum>
     </property>
     <property name="color">
      <color>
       <red>239</red>
       <green>41</green>
       <blue>41</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Sample rate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QRadioButton" name="channelAButton">
     <property name="text">
      <string>Channe&amp;l A</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="14" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="2" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QDoubleSpinBox" name="snrSpin">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> dB</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>20.000000000000000</double>
     </property>
     <property name="value">
      <double>16.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QRadioButton" name="channelBButton">
     <property name="text">
      <string>Channel &amp;B</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="KLed" name="channelBLed">
     <property name="state">
      <enum>KLed::Off</enum>
     </property>
     <property name="look">
      <enum>KLed::Sunken</enum>
     </property>
     <property name="darkFactor">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="KLed" name="channelALed">
     <property name="state">
      <enum>KLed::Off</enum>
     </property>
     <property name="look">
      <enum>KLed::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>&amp;Save</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Channel B</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="sampleRateSpin">
     <property name="text">
      <string>xxx</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Channel A</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="KLed" name="syncLed">
     <property name="state">
      <enum>KLed::Off</enum>
     </property>
     <property name="look">
      <enum>KLed::Sunken</enum>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>223</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="QCheckBox" name="falseColorCheck">
     <property name="text">
      <string>&amp;False color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="15">
    <widget class="QScrollBar" name="offsetScrollBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Syncing...</string>
     </property>
    </widget>
   </item>
   <item row="13" column="4">
    <widget class="QPushButton" name="resetLabel">
     <property name="text">
      <string>&amp;Reset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>SNR</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="15">
    <widget class="QStackedWidget" name="mapStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Waiting for APT sync, please wait...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Carrier</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sync</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QComboBox" name="paletteCombo">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
